2014/20140603__ca__primary__kern__precinct.csv

```diff
@@ -12991,13 +13213,15 @@ Kern,0040443,Secretary of State,,REP,Roy Allmond,8
 Kern,0040443,State Assembly,34,REP,Shannon Grove,125
 Kern,0040443,State Assembly,34,DEM,"Virginia ""Mari"" Goodman",33
 Kern,0040443,State Senate,16,REP,Jean Fuller,127
+Kern,0040443,State Senate,16,DEM,Ruth Musser-Lopez,1
 Kern,0040443,Superintendent of Public Instruction,,NOP,Lydia A. Gutiérrez,24
 Kern,0040443,Superintendent of Public Instruction,,NOP,Marshall Tuck,61
 Kern,0040443,Superintendent of Public Instruction,,NOP,Tom Torlakson,61
 Kern,0040443,Treasurer,,GRN,Ellen H. Brown,6
 Kern,0040443,Treasurer,,REP,Greg Conlon,101
 Kern,0040443,Treasurer,,DEM,John Chiang,44
 Kern,0040443,U.S. House,23,REP,Kevin McCarthy,129
+Kern,0040441,U.S. House,23,DEM,Raul Garcia,1
```

Comment:
Would someone be able to point me to the source for these write-in entries?  I suspect that the precinct here may be incorect.

Reply:
Luckily for my bad memory, I put [the link to my source material](https://web.archive.org/web/20160605145259/https://elections.co.kern.ca.us/elections/SOVC/SOVC_June_3_2014_full.pdf) in [the PR description](https://github.com/openelections/openelections-data-ca/issues/106).

Please fix if you find any errors.

Reply:
Thanks @carbonphyber.  Pull request #235 fixes a tiny error (probably a copy/paste issue).